src/app/config/schema.ts

```diff
@@ -319,6 +319,12 @@ export const optionalVarsSchema: Schema<IOptionalVarsSchema> = {
       default: 100,
       env: 'PUBLIC_API_RATE_LIMIT',
     },
+    platformApi: {
+      doc: 'Per-minute, per-IP, per-instance request limit for platform APIs',
```

Comment:
not within scope of this PR's code, but do consider setting overall CF rate limit for platform api endpoint too

Reply:
Ok I see it is in [FRM-948](https://linear.app/ogp/issue/FRM-948). can just set CF rate limit 😆

Reply:
great point! i've updated the description in FRM-948 to include setting CF rate limit